assets/src/design-system/components/snackbar/stories/index.js

```diff
@@ -41,113 +40,109 @@ export default {
 export const _default = () => (
   <ThemeProvider theme={theme}>
     <Container>
-      <SnackbarContainer>
-        <SnackbarMessage
-          actionLabel={text('actionLabel', '')}
-          aria-label={text(
-            'aria-label',
-            'this is my aria label giving my message context for screen reader users'
-          )}
-          customZIndex={number('customZIndex')}
-          onAction={action('on action clicked')}
-          onDismiss={action('on dismiss fired')}
-          isPreventAutoDismiss={boolean('isPreventAutoDismiss')}
-          message={text('message', 'Sorry! File failed to upload.')}
-        />
-      </SnackbarContainer>
+      <SnackbarContainer
+        notifications={[
+          {
+            actionLabel: text('actionLabel', ''),
+            'aria-label': text(
+              'aria-label',
+              'this is my aria label giving my message context for screen reader users'
+            ),
+            customZIndex: number('customZIndex'),
+            dismissable: boolean('dismissable'),
+            preventAutoDismiss: boolean('preventAutoDismiss'),
+            message: text('message', 'Sorry! File failed to upload.'),
+            onAction: action('on action clicked'),
+            onDismiss: action('on dismiss fired'),
+          },
+        ]}
+      />
```

Comment:
These stories were out of date. I removed the deprecated props and cleaned it up to make sure they rendered.